### ISSUE TITLE
[FEATURE REQUEST] Move ND70 debug print option out of Diagnostic Menu in input.geos

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#73]
The ND70 debug print option is currently included in the Diagnostic Menu of input.geos, however bpch diagnostics and the Diagnostic Menu are on their way out. We should move the debug print option out of this menu. This will also allow users to utilize this option in GCHP (see https://github.com/geoschem/gchp/issues/43).

**Comment by yantosca** [comment]
We will add this into GEOS-Chem 12.7.0.

**Comment by yantosca** [comment]
This has now been implemented in the feature/RemoveBpch branch, which will eventually be merged into 12.7.0.  The menu looks like this now:

```
%%% SIMULATION MENU %%% :
Start YYYYMMDD, hhmmss  : 20160701 000000
End   YYYYMMDD, hhmmss  : 20160701 010000
Run directory           : ./
Root data directory     : /n/holylfs/EXTERNAL_REPOS/GEOS-CHEM/gcgrid/data/ExtData
Met field               : geosfp
Simulation name         : CO2
Turn on debug printout? : F
```
